plugin.yaml

```diff
@@ -132,6 +140,30 @@ node_types:
         poststart:
           implementation: starlingx.cloudify_starlingx.resources.wrcp.poststart
 
+  cloudify.nodes.starlingx.PATCHES_LIST:
+    derived_from: cloudify.nodes.Root
+    properties:
+      <<: *external_resource
+      <<: *client_config
+      resource_config:
+        type:  cloudify.types.starlingx.ClientConfig
```

Comment:
`cloudify.types.starlingx.ClientConfig` is used in line 147, which is where we provided the credentials for communicating with StarlingX API.

Reply:
The question is how to pass ClientConfig and Patch objects into nodes?